src/test/resources/after.pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.pekko</groupId>
+    <artifactId>pekko-stream_3</artifactId>
```

Comment:
* looks good - the key issue is that the artifactId has to be `pekko-stream_3` to match our jar name and previous releases of our jar

Reply:
We shouldn't be messing with either `name` or `artifactId` at all, the entire sbt build one way or another is reliant on it.